polaris.shopify.com/pages/examples/page-full-width.tsx

```diff
@@ -8,15 +8,26 @@ function PageExample() {
     <Page
       fullWidth
       title="Orders"
-      primaryAction={{content: 'Create order', icon: PlusIcon}}
-      secondaryActions={[{content: 'Export'}]}
+      primaryAction={{
+        content: 'Create order',
+        icon: PlusIcon,
+        accessibilityLabel: 'Create order',
+      }}
+      secondaryActions={[
+        {accessibilityLabel: 'Export orders', icon: ExportIcon},
+      ]}
       pagination={{
         hasNext: true,
       }}
     >
-      <LegacyCard title="Credit card" sectioned>
-        <p>Credit card information</p>
-      </LegacyCard>
+      <Card>
+        <Text as="h2" variant="headingSm">
+          Credit card
+        </Text>
+        <Text as="p" variant="bodyMd">
+          Credit card information
+        </Text>
```

Comment:
Card doesn't apply spacing by default for children.
The below mimics the same styling as the previous LegacyCard section and aligns with our Storybook/doc examples.
```suggestion
        <BlockStack gap="200">
          <Text as="h2" variant="headingSm">
            Credit card
          </Text>
          <Text as="p" variant="bodyMd">
            Credit card information
          </Text>
        </BlockStack>
```